published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+# IPFS Weekly #5
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the February sprints:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
```

Comment:
i agree with reducing repetition but the categories dont work as bullets, right now its very easy to get lost in the bullet levels. either make them larger (proper headings) or return to the original `- (topic) message` format, even if repetitive.